python/src/RandomWalkMetropolisHastings_doc.i.in

```diff
@@ -164,22 +164,32 @@ expansionFactor : float, :math:`e > 1`
 // ---------------------------------------------------------------------
 
 %feature("docstring") OT::RandomWalkMetropolisHastings::getAdaptationRange
-"Get the range.
+"Get the expected range for the acceptance rate.
+During burn-in, at the end of every adaptation period,
+if the acceptance rate does not belong to this range,
+the adaptation factor is multiplied
+either by the expansion factor (if the acceptance rate is too high)
+or by the shrink factor (if the acceptance rate is too low).
 
 Returns
 -------
 range : :class:`~openturns.Interval` of dimension 1
-    Range :math:`[m,M]` of the adaptation factor."
+    Range :math:`[m,M]` of the expected acceptance rate."
```

Comment:
In statistics, the lower case latter is used for the observation of a random variable and the upper case letter for a random variable. In linear algebra, the lower case letter is used for a scalar or a vector and the upper case letter for a matrix. Furthermore, I guess that $m$ is the shrinkage factor and $M$ is the expansion factor, but this is not written explicitly. Since this is not what you mean here, I suggest to use full text letters.
```suggestion
    Range :math:`[shrinkFacor, expansionFactor]` of the expected acceptance rate."
```

Reply:
Done